File: Java/백준/Bronze/10987. 모음의 개수/모음의 개수.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        String word = br.readLine();

        int number = 0;

        for (int i = 0; i < word.length(); i++) {
            if (word.charAt(i) == 'a' || word.charAt(i) == 'e' || word.charAt(i) == 'i' || word.charAt(i) == 'o' || word.charAt(i) == 'u') {
                number++;
            }
        }

        System.out.print(number);
    }
}

File: C11/백준/Bronze/10987. 모음의 개수/모음의 개수.c
#include <stdio.h>
#include <string.h>

int main() {
    char word[101];

    scanf("%s", word);

    int number = 0;

    for (int i = 0; i < strlen(word); i++) {
        if (word[i] == 'a' || word[i] == 'e' || word[i] == 'i' || word[i] == 'o' || word[i] == 'u') {
            number++;
        }
    }

    printf("%d", number);

    return 0;
}